src/library/Connect/WorkloadInformationProvider.cs

```diff
@@ -117,6 +122,26 @@ internal class WorkloadInformationProvider : IWorkloadInformationProvider
             }
         }
 
+        public List<int> GetPortsToIgnoreFromAnnotations(V1ServiceList serviceNameSpace) {
+            var portsToIgnore = new List<int>();
+            List<V1Service> portsList = serviceNameSpace.Items.Where(item => item.Metadata?.Annotations?.ContainsKey(ServiceAnnotations) ?? false).ToList();

```

Comment:
Doesn't ContainsKey already return a boolen? why the "?? false"?

Reply:
Cannot implicitly convert type 'bool?' to 'bool'. 
bool? takes in true, false and null. So explicit conversion is required.